packages/ui/form/PhoneInput.tsx

```diff
@@ -18,21 +18,27 @@ export type PhoneInputProps = {
 };
 
 function BasePhoneInput({ name, className = "", onChange, ...rest }: PhoneInputProps) {
+  useEffect(() => {
+    if (!rest.value) {
+      return;
+    }
+    const formattedValue = rest.value.trim().replace(/^\+?/, "+");
```

Comment:
Have done this because it was giving an Invalid phone number whenever I try to submit it without changing the value so separated this logic to add `+` on useEffect instead of onChange. Otherwise, it can be handled via input fields too